pages/settings/activity.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col cols="12" :offset-md="1" md="5" class="pt-0 pb-4">
+        <v-list two-line class="mb-2">
+          <v-list-item-group>
+            <v-list-item
+              v-for="file in logFiles"
+              :key="file.Name"
+              :href="getLogFileLink(file.Name)"
+              target="_blank"
+              rel="noopener"
+            >
+              <v-list-item-avatar>
+                <v-icon>mdi-file</v-icon>
+              </v-list-item-avatar>
+              <v-list-item-content>
+                <v-list-item-title v-text="file.Name" />
+                <v-list-item-subtitle
+                  v-text="getFormattedLogDate(file.DateModified)"
+                />
+              </v-list-item-content>
+              <v-list-item-action>
+                <v-icon>mdi-chevron-right</v-icon>
+              </v-list-item-action>
+            </v-list-item>
+          </v-list-item-group>
+        </v-list>
+      </v-col>
+      <v-col cols="12" md="5" class="pt-0 pb-4">
+        <v-list two-line class="mb-2" disabled>
+          <v-list-item-group>
+            <v-list-item v-for="activity in activityList" :key="activity.Id">
+              <v-list-item-avatar
+                :color="getColorFromSeverity(activity.Severity)"
+              >
+                <v-icon>mdi-shield-account-outline</v-icon>
+              </v-list-item-avatar>
+              <v-list-item-content>
+                <v-list-item-title v-text="activity.Name" />
+                <v-list-item-subtitle v-text="activity.ShortOverview" />
+              </v-list-item-content>
+              <v-list-item-action>
+                <v-list-item-subtitle
+                  class="text-capitalize-first-letter"
+                  v-text="getFormattedActivityDate(activity.Date)"
+                />
+              </v-list-item-action>
+            </v-list-item>
+          </v-list-item-group>
+        </v-list>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapActions } from 'vuex';
+import colors from 'vuetify/lib/util/colors';
+import { ActivityLogEntry, LogFile, LogLevel } from '@jellyfin/client-axios';
+import htmlHelper from '~/mixins/htmlHelper';
+
+export default Vue.extend({
+  mixins: [htmlHelper],
+  middleware: 'adminMiddleware',
+  data() {
+    return {
+      activityList: [] as ActivityLogEntry[],
+      logFiles: [] as LogFile[]
+    };
+  },
+  head() {
+    return {
+      title: this.$store.state.page.title
+    };
+  },
+  async beforeMount() {
+    this.setPageTitle({ title: this.$t('activityLogs') });
+    // calculating date of 7 days ago as min date to get activity logs
+    const minDate = new Date();
+    minDate.setTime(minDate.getTime() - 7 * 24 * 60 * 60 * 1000);
```

Comment:
```suggestion
    minDate.setTime(minDate.getDate() - 7);
```

This should get you exactly 7 days ago without looking so complicated.

Reply:
This should also probably be a computer property.